docs/Course Wiki/PHYS Course Offerings/PHYS212.md

```diff
@@ -21,7 +21,6 @@ PHYS212 is a solid foundational course in many concepts absolutely essential for
 
 ## Prerequisites
 
-[PHYS211][PHYS211.md]
 The official prerequisite for PHYS212 is PHYS211. 
```

Comment:
Add a bullet point for PHYS211 and for MATH241, as it is a corequisite to this course:
- [PHYS211](PHYS211.md)
- [MATH241](../MATH%20Course%20Offerings/MATH241.md) (corequisite)

Also, explain what concepts from PHYS211 and MATH241 are used in PHYS212.